lib/processor/src/girouette/processor.clj

```diff
@@ -157,7 +167,15 @@
         (pp/pprint @file-data file-writer))
       (let [all-css-classes (into #{} (mapcat :css-classes) (vals @file-data))
             predef-garden (if preflight? preflight [])
-            all-garden-defs (into predef-garden (keep (:garden-fn @config)) (sort all-css-classes))]
+            all-garden-defs (into predef-garden (keep (:garden-fn @config)) (sort all-css-classes))
+            all-garden-defs (reduce (fn [garden [cls classes]]
+                                      (into garden
+                                            (util/apply-class-rules
+                                              garden
+                                              (first (normalize-classes cls))
+                                              (set (mapcat normalize-classes classes)))))
+                                    all-garden-defs
+                                    @(requiring-resolve apply-classes))]
```

Comment:
Could you please give an example of how the apply-classes parameter is used?
What kind of values is returned by `@(requiring-resolve apply-classes)`?

Reply:
it is a map from class -> classes, e.g. `{"btn" ["p-2" "rounded"]}`

Reply:
`(requiring-resolve apply-classes)` throws an exception when apply-classes is `nil`. I will fix it, no problem.